### ISSUE TITLE
[RFE] Provide a new option to update the reverse DNS zone in IPA domain

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1836]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1832
- Created at 2013-03-08 23:55:40 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed as Fixed
- Assigned to [jhrozek](https://pagure.io/user/jhrozek)
---

Currently the reverse records are updated automatically using functionality in the bind dyndb plugin. Especially in conjuction with TCP updates described in ticket #1831 we might want to provide a new option that would enable the client to send the PTR record on its own.

We might only permit updating the PTR records over TCP, which would make this RFE depend on ticket #1831.

See also this discussion on freeipa-devel:
https://www.redhat.com/archives/freeipa-devel/2013-March/msg00004.html

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-08 23:56:52](https://pagure.io/SSSD/sssd/issue/1832#233937)**

Fields changed

cc:  => pspacek

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-14 14:43:52](https://pagure.io/SSSD/sssd/issue/1832#233938)**

Fields changed

summary: RFE: Provide a new option to update the reverse DNS zone in IPA domain => [RFE] Provide a new option to update the reverse DNS zone in IPA domain
type: defect => enhancement

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-14 14:46:33](https://pagure.io/SSSD/sssd/issue/1832#233939)**

We need to try to update the PTR record when we do dyndns update against AD as well.
So we need to always optionally try a PTR record update. But ignore any failure, so it needs to be a separate update from the A/AAAA record update.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-15 12:58:23](https://pagure.io/SSSD/sssd/issue/1832#233940)**

We did some basic tests with AD and found that client is able to update A and also PTR records.

A and PTR updates have to be separate because each update targets different zone (forward and reverse), so responsible DNS 'master' server can be different for both zones and each zone can be configured differently.

IMHO it could be nice to have independent enable/disable update option for A/AAAA/PTR records.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-20 18:06:25](https://pagure.io/SSSD/sssd/issue/1832#233941)**

This is a subtask of #1504, I'm already on it.

review:  => 0

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-21 18:38:49](https://pagure.io/SSSD/sssd/issue/1832#233942)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.10 beta

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-03-28 14:35:30](https://pagure.io/SSSD/sssd/issue/1832#233943)**

Linking to the same BZ as #1504

rhbz:  => [https://bugzilla.redhat.com/show_bug.cgi?id=928071 928071]

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-04-17 21:05:35](https://pagure.io/SSSD/sssd/issue/1832#233944)**

Fields changed

owner: somebody => jhrozek
patch: 0 => 1
status: new => assigned

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-05-03 20:50:22](https://pagure.io/SSSD/sssd/issue/1832#233945)**

- master: 38ebc764eeb7693e0c4f0894d6687e54fbba871b

resolution:  => fixed
status: assigned => closed

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-05-30 00:44:34](https://pagure.io/SSSD/sssd/issue/1832#233946)**

Fields changed

design:  => https://fedorahosted.org/sssd/wiki/DesignDocs/ActiveDirectoryDNSUpdates

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2013-05-31 10:15:45](https://pagure.io/SSSD/sssd/issue/1832#233947)**

Fields changed

changelog:  => This option should be mostly left to its default state, no point in advertising to ordinary users.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:48:40](https://pagure.io/SSSD/sssd/issue/1832#233948)**

**Metadata Update from @jhrozek**:
- Issue assigned to jhrozek
- Issue set to the milestone: SSSD 1.10 beta
